### DISCUSSION TITLE
A quick fix for simplified nested interpolated string?

### DISCUSSION THREAD

**Discussion by Flithor** [dotnet/roslyn#65756] [Ideas]
original code:
```
var str1 = "the strings";
var str2 = $"abcd{variable}efg";
var str = $"{str1},{str2}";
```
Now we can quick fix it to
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
```
var str = $"{"the strings"},{$"abcd{variable}efg"}";
```
but we cannot quick fix it to
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
```
var str = $"the strings,abcd{variable}efg";
```
Could you add a quick fix to simplify interpolated string that include const string or other interpolated string?

Or in other words: "Merge constant strings in interpolated string"

**Comment by CyrusNajmabadi** [comment]
It would probably make sense for inline temp to always do this with strings by default.

**Reply by Flithor** [reply]
"Merge constant strings into interpolated string" is optional.
